Tree_main.js

```diff
@@ -79,10 +79,9 @@ var getScriptPromisify = (src) => {
                                 animationDurationUpdate: 750
                             }
                         ]
-                    })
-                );
+                    }
+              chart.setOption(option)
```

Comment:
chart.setOption() function was in incorrect place, so i have moved function inside main function